### ISSUE TITLE
Add ASDC collections to test suite

### ISSUE THREAD

**Issue by danielfromearth** [nsidc/earthaccess#460]

**Comment by asteiker** [comment]
@danielfromearth Are you still interested in adding ASDC collections? Not sure if this has already been tackled since you wrote the ticket.

**Comment by danielfromearth** [comment]
Yes, I'd still like to add ASDC to some of the tests, where ASDC collections are not currently listed, e.g., https://github.com/nsidc/earthaccess/blob/14c4fbeb8f4d27bf9145b558684f7b501214de78/tests/integration/test_cloud_download.py#L15

**Comment by asteiker** [comment]
Thanks for confirming @danielfromearth ! If it only requires an additional dictionary for ASDC, would the good first issue label apply here? I also wonder if we should be adding all providers to this list, or is there a reason we just have a manually curated list?

**Comment by danielfromearth** [comment]
@asteiker - I wonder if there's a way to programmatically get a list of providers, but maybe that could be a different _issue_? And in the meantime, I simply add ASDC as a provider to these dictionaries?
Oh, and sure, the good-first-issue could apply, fine with me!

**Comment by asteiker** [comment]
@danielfromearth Thanks for your feedback! Agreed - I don't want to expand the scope of this issue. It just got me thinking about how we should approach the testing in general. Creating another issue is a good idea.
